core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -467,7 +468,8 @@ private[spark] class TaskSetManager(
   def resourceOffer(
       execId: String,
       host: String,
-      maxLocality: TaskLocality.TaskLocality)
+      maxLocality: TaskLocality.TaskLocality,
+      availableResources: Map[String, Buffer[String]] = Map.empty)
```

Comment:
Could you change the value type to `Seq[String]` since this method shouldn't change it?

Reply:
updated